api/responses/ok.js

```diff
@@ -25,9 +26,9 @@ module.exports = function sendOK(data, options) {
 
   // If appropriate, serve data as JSON(P)
   var JSONresponse = {
-    links: {
+    links: _.extend({
       self: req.baseUrl + req.url
-    },
+    }, links),
```

Comment:
I'll pick up this suggestion in another task because another PR I have raised abstracts the getJSONresponse so it can be stubbed. If I wait for that to be merged I can make use of the abstraction rather than doing it in this pr and have irritating conflicts.